### PR TITLE
feat(at-internet): add missing data (country, currency)

### PR DESCRIPTION
**Pull request by jleveugle**

### CODE REVIEW

**Comment by marie-j** [comment]
Maybe it could make sense to have this as a shared config for all managers

File: client/app/app.js [hunk 1]
```diff
@@ -211,4 +205,17 @@ angular
   .run(($translate) => {
     moment.locale(_.first($translate.use().split('_')));
   })
+  .run((constants, atInternet, OvhApiMe) => {
```
Comment (antleblanc) [inline client/app/app.js]: Explicit annotation would be nice.

```suggestion
  .run(/* @ngInject */(constants, atInternet, OvhApiMe) => {
```

File: client/app/app.js [hunk 2]
```diff
@@ -211,4 +205,17 @@ angular
...
+    const level2 = constants.target === 'US' ? '57' : '10';
```
Comment (antleblanc) [inline client/app/app.js]: Theses values shouldn't be moved into a constant file? In order to facilitate any re-use.

File: client/app/app.js [hunk 3]
```diff
@@ -211,4 +205,17 @@ angular
...
+
+    OvhApiMe.v6().get().$promise
```
Comment (antleblanc) [inline client/app/app.js]: No `return` statement?

**Comment by antleblanc** [comment]
No internal ticket referenced.

Is it a hotfix?

Please provide the bare minimum information!

Thanks!

**Pull request merged**